runtimes/eden/Cargo.toml

```diff
@@ -37,7 +37,7 @@ std = [
   "pallet-uniques/std",
   "pallet-utility/std",
   "pallet-aura/std",
-  "pallet-xcm/std",
+  #"pallet-xcm/std",
```

Comment:
If we could remove the commented lines in this file, that would be neater in my opinion.